pkg/derivation/encode.go

```diff
@@ -19,6 +18,16 @@ var stringEscaper = strings.NewReplacer(
 	"\"", "\\\"",
 )
 
+// nolint:gochecknoglobals
+var (
+	comma        = []byte{','}
```

Comment:
```suggestion
const (
	comma        = `,`
```
I wonder if there are performance improvements the compiler can find. If it knows that the value is immutable. This would mean casting the character to []byte on each usage though.